packages/patternfly-4/react-core/src/components/Select/Select.test.tsx

```diff
@@ -6,6 +6,7 @@ import { CheckboxSelectOption } from './CheckboxSelectOption';
 import { SelectGroup } from './SelectGroup';
 import { CheckboxSelectGroup } from './CheckboxSelectGroup';
 import { SelectVariant } from './selectConstants';
+import { SelectContext } from '../../../dist/js';
```

Comment:
tests shouldn't require the dist to be built, no? can you load it from source?

Reply:
Yes, tests don't need the `dist`. Jest builds them itself according to our rules.